gem/static/css/_style.surveys.scss

```diff
@@ -45,6 +45,85 @@
             @extend %surveys__checkbox;
         }
     }
+
+    &__item + &__item {
+        padding-top: 24px;
+        margin-top: 24px;
+        border-top: 1px solid #ffffff;
+    }
+
+    &__tout {
+        background-color: $colorFirst;
+        color: #ffffff;
+        margin: 0 0 24px 0;
+        padding: 0 16px;
+        text-align: center;
+
+        .section-title {
+            text-align: left;
+
+          h2 {
+            height: 30px;
+            line-height: 30px;
+            margin: 0 0 24px 0;
+            font-family: 'AvenirNextCondensed-Medium', sans-serif;
```

Comment:
We can use a SASS var here much like `font-family: $font-family-avenirBold;`but not hte end of the world